### ISSUE TITLE
RequestId mismatch - expected, actual

### ISSUE THREAD

**Issue by Kotodian** [k-sone/snmpgo#25]
why some times report the error 
![image](https://user-images.githubusercontent.com/22030659/87137397-67941c00-c2cf-11ea-90ce-6cb5bdee173c.png)

**Comment by k-sone** [comment]
This is all I can think of right now.

1. You use an instance of `snmpgo.SNMP` from multiple goroutines
    - `snmpgo.SNMP` is not goroutine-safe
2. Network problem
    - Packets are lost or out of order
3. SNMP agent problem
    - something is wrong with SNMP agent

**Comment by Kotodian** [comment]
yeah,I use snmpgo.SNMP in crontab to get the hardware's information.how to deal with the problem
